Berksfile

```diff
@@ -3,6 +3,6 @@ source "https://supermarket.getchef.com"
 metadata
 
 group :integration do
-  cookbook 'selinux', '~> 0.9.0'
+  cookbook 'selinux', git: 'https://github.com/jbartko/selinux.git', branch: 'feature/add-serverspec'
```

Comment:
The dependent selinux cookbook has broken Fedora support. The git branch corresponds to PR skottler/selinux#28 which adds serverspec tests and updates Fedora support.

If maintaining the dependency is valuable for the purpose of integration testing this cookbook, could you consider commenting on the PR? I'm not sure how attentive the maintainer is and wanted to give them an opportunity to review and merge changes instead of creating yet another fork. 

In the meantime, I will attempt to update this PR without git refs in the Berksfile.